rootston/desktop.c

```diff
@@ -233,6 +243,7 @@ void view_set_fullscreen(struct roots_view *view, bool fullscreen,
 		view->saved.x = view->x;
 		view->saved.y = view->y;
 		view->saved.rotation = view->rotation;
+		view->saved.alpha = view->alpha;
```

Comment:
Is that every read?

It see a reason to set to full opacity for fullscreen, but that doesn't seem to be implemented.